sql/moz-fx-data-shared-prod/monitoring_derived/glean_server_knob_experiments_v1/metadata.yaml

```diff
@@ -0,0 +1,12 @@
+friendly_name: Glean Server Knobs
+description: |-
+  A view on the experiment monitoring dataset of all glean server knob configs
+labels:
+  incremental: false
+  public_bigquery: false
+  public_json: false
```

Comment:
you mentioned you want to make this data public, should any of this be set to `true`, or is this something for later?

Reply:
yeah that's the intention eventually just wanted to make sure the derived table worked as expected.